docs/core/cwc-tag.md

```diff
@@ -0,0 +1,9 @@
+# cwc-tag
```

Comment:
Can you add two links to the tag and badge components to this file?

https://github.com/vmware/clarity/blob/master/docs/core/README.md

Reply:
Yup will do